README.md

```diff
@@ -50,6 +50,18 @@ A spring bean:
 
    }
 ``` 
+## Configuration for Service Authentication filter
+The following values must be provided to enable a ServiceAuthFilter bean:
+```yaml
+idam:
+  s2s-authorised:
+    services: microservice1, microservice2
+```
+ServiceAuthFilter bean is OncePerRequestFilter filter that you can add to your filter chain to authorise service 
+request. The filter will expect 'ServiceAuthorization' Bearer token as part of the request header that it will consume 
```

Comment:
```suggestion
request. The filter will expect a header with '`ServiceAuthorization: Bearer <token>`' as part of the request header that it will consume 
```